packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -290,16 +295,8 @@ export default class TSGenerator extends CodeGeneratorLanguage {
         docs: [
           {
             description: writer =>
-              writer.writeLine(
-                wordWrap('Optionally configure various options, such as response parsing, that the SDK allows.')
-              ),
-            tags: [
-              { tagName: 'param', text: 'config Object of supported SDK options and toggles.' },
-              {
-                tagName: 'param',
-                text: 'config.parseResponse If responses are parsed according to its `Content-Type` header.',
-              },
-            ],
+              writer.writeLine(wordWrap('Optionally configure various options that the SDK allows.')),
+            tags: [{ tagName: 'param', text: 'config Object of supported SDK options and toggles.' }],
```

Comment:
Though we aren't using `config.parseResponse` anymore I'm keeping the entire config system in place everywhere because I'll need it to support custom request timeouts in https://github.com/readmeio/api/issues/432.